netty/build.gradle

```diff
@@ -17,20 +17,21 @@ evaluationDependsOn(project(':grpc-core').path)
 
 dependencies {
     api project(':grpc-core'),
-            libraries.netty,
-            libraries.netty_epoll
+            libraries.netty
     implementation libraries.netty_proxy_handler,
             libraries.guava,
             libraries.errorprone,
-            libraries.perfmark
+            libraries.perfmark,
+            ("io.netty:netty-transport-native-epoll:${nettyVersion}")
```

Comment:
Remove this dependency. Everything in epoll is accessed via reflection. (Also, dependencies like this should be in `libraries` and referenced here.)

Reply:
But without this I get errors like

```
UdsNameResolverProvider.java:23: error: package io.netty.channel.unix does not exist
import io.netty.channel.unix.DomainSocketAddress;
                            ^
UdsNameResolver.java:25: error: package io.netty.channel.unix does not exist
import io.netty.channel.unix.DomainSocketAddress;
                            ^
UdsNettyChannelProvider.java:25: error: package io.netty.channel.unix does not exist
import io.netty.channel.unix.DomainSocketAddress;
                            ^
UdsNameResolverProvider.java:69: error: cannot find symbol
    return Collections.singleton(DomainSocketAddress.class);

```

I can add it to libraries and reference it here but I cannot remove the dependency unless I am missing something?

Reply:
> DomainSocketAddress is part of netty-transport-native-unix-common.

Reply:
I think I have gotten it right now.